frontend/src/components/CsetComparisonPage.jsx

```diff
@@ -25,6 +25,21 @@ import {useDataCache} from "../state/DataCache";
 import {getResearcherIdsFromCsets, useDataGetter} from "../state/DataGetter";
 import {LI} from "./AboutPage";
 
+const makeGraph = (edges) => {
```

Comment:
## Add: `makeGraph()`
I needed to add this because we didn't push our changes earlier. But I can `rebase` from #691 when we push it.